packages/agent-utils/src/agent/AgentVariables.ts

```diff
@@ -5,7 +5,7 @@ export class AgentVariables {
   public static Prefix = "${";
   public static Suffix = "}";
 
-  constructor(private _saveThreshold: number = 2000) { }
+  constructor(private _saveThreshold: number = 3000) { }
```

Comment:
Using 3000 yielded some rough results for me, it lead to more summarizations (which still is pretty naive in its implementation). Are you good with using 2000 for now, or do you have specific reasons for using 3000?

Reply:
I found 3000 to work better in the past, but I'm good with 2000 cause haven't noticed a notable difference recently